bundles/http/Citrix/DocuSign/73f4e3b3-e2f3-43b3-9ce6-7f12da7b714c.js

```diff
@@ -0,0 +1,660 @@
+                ]
+            },
+            {
+                name: "templates_documents",
+                columns: [
+                    { name: "document_id", type: "INTEGER" },
+                    { name: "name", type: "STRING", length: 255 },
+                    { name: "parent_template_id", type: "STRING", length: 255 },
+                    { name: "unique_id", type: "STRING", primaryKey: true, length: 255 },
+                    { name: "root_template_id", type: "STRING", length: 255 },
+                ]
+            },
+            {
+                name: "templates_folder_ids",
+                columns: [
+                    { name: "value", type: "STRING", length: 255 },
+                    { name: "parent_template_id", type: "STRING", length: 255 },
+                    { name: "unique_id", type: "STRING", primaryKey: true, length: 255 },
+                    { name: "root_template_id", type: "STRING", length: 255 },
+                ]
+            },
+            {
+                name: "users",
+                columns: [
+                    { name: "created_date_time", type: "DATETIME" },
+                    { name: "email", type: "STRING", length: 255 },
+                    { name: "first_name", type: "STRING", length: 255 },
+                    { name: "job_title", type: "STRING", length: 255 },
+                    { name: "last_name", type: "STRING", length: 255 },
+                    { name: "user_id", type: "STRING", length: 255, primaryKey: true },
+                    { name: "user_name", type: "STRING", length: 255 },
+                    { name: "user_status", type: "STRING", length: 255 },
+                    { name: "user_type", type: "STRING", length: 255 },
+                ]
+            }
+        ],
+        relationships: [
+            {
+                name: "nested_table_1",
+                primaryTable: "templates",
+                foreignTable: "templates_documents",
+                columnPairs: [
+                    {
+                        primaryKey: "template_id",
+                        foreignKey: "parent_template_id"
+                    }
+                ]
+            },
+            {
+                name: "nested_table_2",
+                primaryTable: "templates",
+                foreignTable: "templates_folder_ids",
+                columnPairs: [
+                    {
+                        primaryKey: "template_id",
+                        foreignKey: "parent_template_id"
+                    }
+                ]
+            },
+            {
+                name: "nested_table_3",
+                primaryTable: "envelopes",
+                foreignTable: "envelopes_recipients_signers",
+                columnPairs: [
+                    {
+                        primaryKey: "envelope_id",
+                        foreignKey: "parent_envelope_id"
+                    }
+                ]
+            }
+        ]
+    },
+    actions: [
```

Comment:
You should consider setting up relevant attributes to required, so that you don't run into a problem that the parameter won't come from the microapp.

Reply:
changes implemented